resteasy-core/src/main/java/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -199,6 +213,11 @@ public ClassLoader run() {
       }
    }
 
+   public boolean outputStreamWasWrittenTo()
```

Comment:
This should be package private or `protected` if we feel this type will be extended.

Reply:
Done.